src/Framework/OpenXmlBasePart.cs

```diff
@@ -81,6 +81,11 @@ internal void Load(OpenXmlPackage openXmlPackage, OpenXmlPart parent, Uri uriTar
             this._uri = uriTarget;
 
             // TODO: should we delay load?
+            if (!this.OpenXmlPackage.Package.PartExists(uriTarget))
+            {
+                throw new OpenXmlPackageException(string.Format("The part with URI target {0} does not exist in the package.", uriTarget));
+            }
+
             PackagePart metroPart = this.OpenXmlPackage.Package.GetPart(uriTarget);
```

Comment:
Because you can't otherwise see this in the conversation, here is essentially the line that should be wrapped in a try-catch statement if you did not fix the exception message in `GetPart()` itself.